react/modules/sample/src/configs/FormComposerConfig.js

```diff
@@ -0,0 +1,644 @@
+export const FormComposerConfig=[
```

Comment:
Remove uneccessary code and keep it simple and clean, People will refer this config to implement right. Also add comments to wherever you have made changes for boundary filter in this config